src/org/zaproxy/zap/extension/httpsessions/ExtensionHttpSessions.java

```diff
@@ -336,6 +350,17 @@ private void unmarkRemovedDefaultSessionToken(String site, String token) {
 		removed.remove(token);
 	}
 
+	private ExtensionSearch getExtensionSearch() {
+		if (extSearch == null) {
+			extSearch = Control.getSingleton().getExtensionLoader().getExtension(ExtensionSearch.class);
+		}
+		return extSearch;
+	}
+
+	public void search (Pattern p) {
+		getExtensionSearch().search(p.pattern(), ExtensionSearch.Type.All, true, false);
```

Comment:
This will lead to NPE if the extension is disabled.

Reply:
Added a check for Extension Search enabled in isEnabledForComponent.